LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014, Matthias Vallentin
+Copyright (c) 2014, Tenzir GmbH
```

Comment:
Please make this a separate commit. License changes should be very minimalistic and "side-effect-free".